ob-chatgpt-shell.el

```diff
@@ -118,12 +132,22 @@ This function is called by `org-babel-execute-src-block'"
                            (string-trim (org-element-property :value element)))
                    'language (when (org-element-property :language element)
                                (string-trim (org-element-property :language element)))
+                   'parameters (when (org-element-property :parameters element)
+                                 (ob-chatgpt--string-to-plist
+                                  (string-trim (org-element-property :parameters element))))
                    'result (save-restriction
                              (goto-char (org-element-property :begin element))
                              (when (org-babel-where-is-src-block-result)
                                (goto-char (org-babel-where-is-src-block-result))
                                (org-babel-read-result)))))))))
 
+(defun ob-chatgpt--string-to-plist (str)
```

Comment:
We can use `org-babel-parse-header-arguments` for this.

Reply:
Ah great to know!